go/otgrpc/client.go

```diff
@@ -72,8 +72,8 @@ func OpenTracingClientInterceptor(tracer opentracing.Tracer, optFuncs ...Option)
 				clientSpan.LogFields(log.Object("gRPC response", resp))
 			}
 		} else {
-			clientSpan.LogFields(log.String("event", "error"), log.String("message", err.Error()))
-			ext.Error.Set(clientSpan, true)
+			SetSpanTags(clientSpan, err, true)
+      clientSpan.LogFields(log.String("event", "error"), log.String("message", err.Error()))
```

Comment:
oh yeah sorry about that, I think it was me resolving conflicts via the Github web UI and it messed up the indentation, thanks for catching and fixing @annismckenzie and @bhs !